### DISCUSSION TITLE
Saleor service keeps restarting and reports ModuleNotFound

### DISCUSSION THREAD

**Discussion by cenekSvoboda** [saleor/saleor#8904] [Deployment]
version:
saleor-2.11.10

problem:
I have successfully started production dashboard and storefront, but when I run django-backend I have issues.
When I run "docker-compose down", "docker system prune --volumes", "docker-compose build" and "docker-compose up" I get many ModuleNotFound errors. The system also tells me that it runs in development mode, despite the fact that I've set DEBUG=False in multiple places and as an environment variable. 

Can you help?

**Comment by karolkielecki** [comment]
Hi! What would you like to achieve? Are you running [the saleor-platform repo](https://github.com/saleor/saleor-platform)? Could you please share some screenshots?

**Reply by cenekSvoboda** [reply]
services worker and api randomly restart...
![image](https://user-images.githubusercontent.com/5390335/148536018-e46fa6d2-c768-41b8-9184-3e03dbac9aa7.png)

**Reply by cenekSvoboda** [reply]
Which repository is meant to be a standalone production one?

**Reply by karolkielecki** [reply]
We recommend using Saleor 3.0 beta for new projects. There are three components:
- Core API: https://github.com/saleor/saleor
- Saleor Dashboard: https://github.com/saleor/saleor-dashboard
- Saleor Storefront: https://github.com/saleor/saleor-storefront, but it's deprecated (we don't maintain it anymore).  We're working on a new project: https://github.com/saleor/react-storefront. 

The [saleor-platform](https://github.com/saleor/saleor-platform) repo hasn't been designed for production purposes, only for development.

**Reply by cenekSvoboda** [reply]
When will the project get out of beta stage?

**Reply by karolkielecki** [reply]
There is no ETA, but the latest beta version is much improved than 2.11.

**Reply by cenekSvoboda** [reply]
OK Karol. Thank you, if I'll encounter some problems, I'll put them here.

**Reply by cenekSvoboda** [reply]
First, I must say, that these three repositories (saleor, saleor-dashboard and react-storefront) are not under docker-compose and there's no step-by-step doc how to run them.

**Reply by karolkielecki** [reply]
Regarding docs--we're aware they're not perfect. We'll improve it. You can find more info on how to run them in the readme sections on the GitHub pages and our docs:
- https://github.com/saleor/saleor-dashboard#getting-started
- https://docs.saleor.io/docs/3.0/developer/running-saleor/configuration

**Reply by cenekSvoboda** [reply]
Please help me debug why the api and worker restarts

**Comment by cenekSvoboda** [comment]
I have this: https://github.com/saleor/saleor/releases/tag/2.11.10
